Emby.Server.Implementations/Updates/InstallationManager.cs

```diff
@@ -166,41 +172,10 @@ private static string ValueOrDefault(string str, string def)
             string packageType = null,
             Version applicationVersion = null)
         {
-            if (withRegistration)
-            {
-                var data = new Dictionary<string, string>
-                {
-                    { "key", _securityManager.SupporterKey },
-                    { "mac", _applicationHost.SystemId },
-                    { "systemid", _applicationHost.SystemId }
-                };
-
-                var options = new HttpRequestOptions
-                {
-                    Url = "https://www.mb3admin.local/admin/service/package/retrieveall?includeAllRuntimes=true",
-                    CancellationToken = cancellationToken
-                };
-
-                options.SetPostData(data);
-
-                using (var response = await _httpClient.SendAsync(options, "POST").ConfigureAwait(false))
-                {
-                    using (var json = response.Content)
-                    {
-                        cancellationToken.ThrowIfCancellationRequested();
+            // TODO cvium
```

Comment:
Let's put it like
```suggestion
            // TODO when plugins get back this would need to be fixed
```

Reply:
With the amount of todos in the code adding your name in it isn’t a bad idea.

Reply:
As long as you use it as a signature _along_ with the actual text of "what to do" ;)